README.md

```diff
@@ -27,17 +34,23 @@ Don't forget to use a strong, randomly generated password instead of `<PASSWORD>
 * Fill out the application settings (see [configuration section](#configuration))
 * Click on "Deploy"
 
+#### VPC configuration
+
+If your Exasol database runs on AWS and in a VPC you should run this adapter in the same VPC. By that it can access the Exasol database using a local IP address.
```

Comment:
```suggestion
You should run CloudWatch adapter in the same AWS VPC that runs Exasol database. By that it can access the Exasol database using a local IP address.
```

Reply:
```suggestion
You should run CloudWatch adapter in the same AWS VPC that runs Exasol database. By that it can access the Exasol database using a local IP address.
If your database does not run on AWS you can skip this step an access your Exasol using its public IP address.
```